community/contributing/documentation_guidelines.rst

```diff
@@ -1,91 +1,120 @@
+   tooltip "Edit the file in a fork of this project".
 
-3. Complete all the edits you want to make for that page.
+3. Edit the text in the text editor.
 
-4. Summarize the changes you made in the form at the bottom of the page and
-   click the button labelled **Propose file change** when done.
+4. At the bottom of the web page, summarize the changes you made and click the
+   button **Propose file change**.
 
 5. On the following screens, click the **Create pull request** button until you
-   see a message like *Username wants to merge 1 commit into
-   godotengine:master from Username:patch-6*.
-
-6. A reviewer will evaluate your changes and incorporate them into the docs if
-   they're acceptable. You might also be asked to make
-   modifications to your changes before they're included.
-
-What makes good documentation?
-------------------------------
+   see a message like *Username wants to merge 1 commit into godotengine:master
+   from Username:patch-6*.
 
-Documentation should be well written in plain English, using well-formed
-sentences and various levels of sections and subsections. It should be clear
-and objective. Also, have a look at the :ref:`doc_docs_writing_guidelines`.
+Another contributor will review your changes and merge them into the docs if
+they're good. They may also make changes or ask you to do so before merging.
 
-We differentiate tutorial pages from other documentation pages by these
-definitions:
+Adding new pages
+----------------
 
--  Tutorial: a page aiming at explaining how to use one or more concepts in
-   the editor or scripts in order to achieve a specific goal with a learning
-   purpose (e.g. "Making a simple 2d Pong game", "Applying forces to an
-   object").
--  Documentation: a page describing precisely one and only one concept at a
-   time, if possible exhaustively (e.g. the list of methods of the
-   Sprite class, or an overview of the input management in Godot).
+To add a new page, create a .rst file with a meaningful name in the section you
+want to add a file to, e.g. \ ``tutorials/3d/light_baking.rst``. Write its
+content like you would do for any other file, and make sure to define a
+reference name for Sphinx at the beginning of the file (check other files for
+the syntax), based on the file name with a “doc\_” prefix (e.g. ``..
+_doc_light_baking:``).
 
-You are free to write the kind of documentation you wish, as long as you
-respect the following rules (and the ones on the repo).
+You should then add your page to the relevant “toctree” (table of contents,
+e.g. \ ``tutorials/3d/index.rst``). By convention, the files used to define the
+various levels of toctree are prefixed with an underscore, so in the above
+example the file should be referenced in ``tutorials/3d/_3d_graphics.rst``. Add
+your new filename to the list on a new line, using a relative path and no
+extension, e.g. here ``light_baking``.
 
 Titles
-------
+~~~~~~
```

Comment:
This section should likely clarify that the reference of the page should match its filename as well as its title. (At least as far as possible - sometimes we change page titles and renaming the file is not convenient as it requires directs.)

Though this PR is a good example of title changes and content splits that would warrant also renaming the files and references.